plugins/nf-schema/src/main/nextflow/validation/SchemaValidator.groovy

```diff
@@ -204,7 +204,7 @@ class SchemaValidator extends PluginExtensionPoint {
 
         final target = CH.createBy(source)
         final next = {
-            if(!(it instanceof String || it instanceof Path)) {
+            if(!(it instanceof String || it instanceof Path || it instanceof GString)) {
```

Comment:
You can do `it instanceof CharSequence` to check if a string is a string or GString

Reply:
Ooh that's nice! Thanks I'll update the code :)